mimic/test/test_server_images.py

```diff
@@ -87,9 +89,9 @@ def test_get_image_list_with_details(self):
         for each_image in image_list:
             self.assertEqual(
                 sorted(each_image.keys()),
-                sorted(['id', 'name', 'links', 'status', 'progress', 'minRam',
-                        'OS-DCF:diskConfig', 'OS-EXT-IMG-SIZE:size', 'metadata',
-                        'minDisk', 'created', 'updated']))
+                sorted(['id', 'name', 'links', 'minRam',
```

Comment:
Change this test class to load IAD instead of ORD so that this validation applies to onmetal images as well.

Also, just adding IAD to the list of regions will not help cause currently the APIMockHelper just fetched the endpoint for the first region :/